### PR TITLE
Vmal

### PR DESCRIPTION
**Pull request by Malisevych**
Add localization to product pages and image pages
Add front-end validation during uploading and updating image

### CODE REVIEW

File: MVCapplication/src/main/java/com/softserve/if072/mvcapp/service/ImageUploadService.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        image.setFileName(multipartFile.getOriginalFilename());
+        image.setImageData(multipartFile.getBytes());
+
+        final String uri = imageUrl + "/";
+
+        Product product = productPageService.getProduct(productId);
+
+        image.setId(product.getImage().getId());
+
+        //HttpEntity<Image> requestEntity = new HttpEntity<>(image);
```
Comment (kucherif) [inline MVCapplication/src/main/java/com/softserve/if072/mvcapp/service/ImageUploadService.java:98]: remove

File: MVCapplication/src/main/java/com/softserve/if072/mvcapp/service/ProductPageService.java [hunk 2]
```diff
@@ -180,6 +216,13 @@ public void delProduct(int productId) {
         restTemplate.delete(uri, param);
     }
 
+    /**
+     * This method sends userId to the RESTful service to receive all stores of user
+     *
+     * @param userId user whose stores must be received
+     * @return list of stores
+     */
+
```
Comment (kucherif) [inline MVCapplication/src/main/java/com/softserve/if072/mvcapp/service/ProductPageService.java:225]: space is not needed

File: MVCapplication/src/main/webapp/WEB-INF/views/product/editProduct.jsp [hunk 3]
```diff
@@ -29,36 +30,52 @@
...
-                        <sf:errors path="name" cssClass="form-control label-danger"/>
+                        <label class="control-label" for="inputWarning">
+                            <spring:message code="product.productName"/>
+                        </label>
                         <sf:input path="name" class="form-control" id="inputWarning"
                                   placeholder="Product name" type="text"/>
                     </div>
                     <div class="form-group">
-                        <label class="control-label">Product description</label>
-                        <sf:errors path="description" cssClass="form-control label-danger"/>
```
Comment (kucherif) [inline MVCapplication/src/main/webapp/WEB-INF/views/product/editProduct.jsp:40]: Is this row not needed?

Comment (Malisevych) [reply MVCapplication/src/main/webapp/WEB-INF/views/product/editProduct.jsp:40]: I decided to print the error messages validation in one block above input fields

**Pull request merged**